packages/payment/src/services/payment-module.ts

```diff
@@ -567,7 +567,10 @@ export default class PaymentModuleService<
     )
 
     // When the entire authorized amount has been captured, we mark it fully capture by setting the captured_at field
-    if (capturedAmount.plus(newCaptureAmount).eq(authorizedAmount)) {
+    const totalCaptured = MathBN.convert(
+      MathBN.add(capturedAmount, newCaptureAmount)
+    )
+    if (MathBN.eq(totalCaptured, authorizedAmount)) {
```

Comment:
Q: just to be sure, even if that is not suppose the happen, would gte be ok to be used here?

Reply:
fair enough... will add it.